app/js/track_events.js

```diff
@@ -0,0 +1,42 @@
+$(function() {
+  'use strict';
+
+  trackMenuClick();
+
+  analytics.trackForm($('.Nav form'), 'Subscribed Newsletter in the menu', getFormProperties);
+  analytics.trackForm($('.Newsletter form'), 'Subscribed Newsletter in the main content', getFormProperties);
+
+  $('[data-track-link]').each(function() {
+    var $element = $(this);
+    analytics.trackLink($element, $element.data('trackLink'));
+  });
+
+  $('[data-track-link-mailto]').each(function() {
+    var $element = $(this);
+    analytics.trackLink($element, $element.data('trackLinkMailto'), getMailFromMailTo);
+  });
+
+  function trackMenuClick() {
+    $('#BurgerBtn').on('click', function(event) {
+      var $btn = $(event.target);
+      if ($btn.hasClass('BurgerBtn--open')) {
+        analytics.track('Opened main menu');
+      } else {
+        analytics.track('Closed main menu');
+      }
```

Comment:
the event that is setting the `BurgerBtn--open` class is also a click on this element. is there any way to guarantee this one always happens later?
Otherwise we might be tracking the events in reverse